docs/content/developers/writing-style-guide.md

```diff
@@ -0,0 +1,222 @@
+
+### Use Correct Proper Nouns
+
+#### DDEV != `ddev`
+
+DDEV is a product and `ddev` is a binary or console command. DDEV should always be uppercase, and `ddev` should always be in backticks. DDEV-Local and DDEV-Live are former product incarnations that shouldn’t be found in modern documentation.
+
+| Write This 👍 | Not This ❌
+| -- | --
+| DDEV is wonderful! | Ddev is wonderful!<br>ddev is wonderful!<br>DDEV-Local is wonderful!
+| Run `ddev`. | Run ddev.<br>Run DDEV.
+
+#### Products, Organizations, and Protocols
+
+When in doubt, honor whatever name a product or organization uses in its official materials.  
+Use backticks to differentiate between a product and command, just like DDEV vs. `ddev`.
+
+| Write This 👍 | Not This ❌
+| -- | --
+| AMD64, ARM64, and Apple Silicon | amd64, arm64, M1 Macs
+| Apache | apache
+| Bash or `bash` | bash
+| Blackfire | blackfire, Blackfire.io
+| Chocolatey | chocolatey
+| Colima or `colima` | colima
+| Composer or `composer` | composer
+| Docker or `docker` | docker
+| Drush or `drush` | drush
+| Git | git
+| Git Bash | git bash
+| GitHub or `github` | Github
+| Gitpod or `gitpod` | Gitpod.io, GitPod
+| GoLand | Goland
+| Google | google
+| Homebrew | homebrew
+| HTTP, HTTPS, SSH, `http`, `https`, `ssh` | http, https, ssh
+| IPv4, IPv6 | IPV4, IPV6, ipv4, ipv6
+| Linux | linux
+| nginx or `nginx` | Nginx, NGINX
+| Node.js or `node` | Node, node
+| NFS | nfs
+| Pantheon | pantheon, Pantheon.io
+| PHP or `php` | php
+| PhpStorm | PHPStorm, PHPstorm, Phpstorm
+| PHPUnit or `phpunit` | phpunit, PHPunit
+| PostgreSQL | Postgres
+| Terminus | terminus
+| Windows | windows
+| Xdebug | XDebug, xDebug
+
+### Quote Copied Text
+
+If you’re quoting a human being or a message lifted verbatim from some other source (outside a fenced code block), make sure it ends up in a `<blockquote>` element:
+
+```
+> Error: your quote style should not always be in a fenced block.
+```
+
+> Error: your quote style should not always be in a fenced block.
+
+### Other Recommendations
+
+One-off tips that don’t fit nicely into any of the sections above:
+
+- Pluralize “CMS” as “CMSes”, not “CMSs”.
+- Use all-caps references for file *types* like JSON, YAML and CSS.
+- Wrap file *extensions* in backticks like `.json`, `.yaml`, and `.css`.
+- Wrap references to files, directories, images and commands in backticks.
+- Use Title Case for headings wherever it makes sense.
+- Link to related services and topics where convenient—usually first use on a given page.
+- Use `<kbd>` elements for representing literal keystrokes.
+- Use sequential numbers for numbered lists in the source Markdown, regardless of how they’re eventually rendered.
+- Try to maintain parallel format for list items.
+
+| Write This 👍 | Not This ❌ |
+| -- | -- |
+| web server | webserver
```

Comment:
I like webserver better :)

Reply:
You’re clearly not alone! Same issue in [this ancient battleground](https://code.djangoproject.com/ticket/20330).

Reply:
I can live with styles I don't naturally do.